From 1fad43cb901a84f62647ac4b72ad8394e5dd16b5 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 16 Jan 2021 10:46:20 +0100
Subject: [PATCH 1/1] dm: core: describe uclass_root_s

make htmldocs creates a warning:

./include/asm-generic/global_data.h:443:
warning: Function parameter or member 'uclass_root_s'
not described in 'global_data'

Correct the member descriptions.

Fixes: 8a715530bb1f ("dm: core: Allow the uclass list to move")
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 include/asm-generic/global_data.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/include/asm-generic/global_data.h b/include/asm-generic/global_data.h
index efa09a1943..9fa9a17003 100644
--- a/include/asm-generic/global_data.h
+++ b/include/asm-generic/global_data.h
@@ -198,13 +198,13 @@ struct global_data {
 	 */
 	struct udevice *dm_root_f;
 	/**
-	 * @uclass_root: head of core tree
+	 * @uclass_root_s: static head of core tree, if uclasses are in
+	 * read-only memory and cannot be adjusted to use @uclass_root as a
+	 * list head.
 	 */
 	struct list_head uclass_root_s;
 	/**
-	 * @uclass_root: pointer to head of core tree, if uclasses are in
-	 * read-only memory and cannot be adjusted to use @uclass_root as a
-	 * list head.
+	 * @uclass_root: pointer to the head of core tree
 	 */
 	struct list_head *uclass_root;
 # if CONFIG_IS_ENABLED(OF_PLATDATA)
-- 
2.29.2

